src/containers/Layout/store.ts

```diff
@@ -0,0 +1,22 @@
+import { createStoreSelectors } from 'utilities';
+import { create } from 'zustand';
+import { persist } from 'zustand/middleware';
+
+interface State {
+  isScrollToTopVisible: boolean;
+  setScrollToTopVisible: (isScrollToTopVisible: State['isScrollToTopVisible']) => void;
+}
+
+const useStore = create<State>()(
+  persist(
+    set => ({
+      isScrollToTopVisible: false,
+      setScrollToTopVisible: (v: boolean) => set(() => ({ isScrollToTopVisible: v })),
+    }),
+    {
+      name: 'scroll-to-top-visibility',
+    },
+  ),
```

Comment:
The `persist` function is used to save the store in local storage so that its state persist between sessions. In this case I think we should remove it as this doesn't need to be persisted.
```suggestion
    set => ({
      isScrollToTopVisible: false,
      setScrollToTopVisible: (v: boolean) => set(() => ({ isScrollToTopVisible: v })),
    }),
```